appengine-java8/endpoints-v2-skeleton/src/main/java/com/example/skeleton/MyApi.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not  use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.example.skeleton;
+
+import com.google.api.server.spi.config.Api;
+
+/**
+ * MyApi skeleton endpoints sample
+ * Add your first API methods in this class, or you may create another class.
+ * In that case, update your src/main/webapp/WEB-INF/web.xml accordingly.
```

Comment:
"In that case" comment, can  you rephrase that to, if you create another class ?